### PR TITLE
Update required labels requirements in dockerfile

### PR DESCRIPTION
**Pull request by rcerven**
* OSBS-7697

Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: docs/users.rst [hunk 1]
```diff
@@ -212,13 +212,16 @@ Some labels are required to be defined:
...
+  The value can't be empty.
 - ``name``: value is used to define the repository name in container registry to
   push built image. Limit this to lowercase alphanumerical values with the
   possibility to use dash as a word separator. A single ``/`` is also allowed.
   ``.`` is not allowed in the first section. For instance, **fed/rsys.log** and
   **rsyslog** are allowed, but **fe.d/rsyslog** and **rsys.log** aren't.
+  The value can't be empty.
 - ``version``: used as the version portion of Koji build NVR, as well as, for
   the version tag in container repository.
+  The value can't be empty, and maybe defined via ENV variable from parent.
```
Comment (athos-ribeiro) [inline docs/users.rst]: s/maybe/may be/

File: docs/users.rst [hunk 2]
```diff
@@ -232,7 +235,8 @@ value.  The container image will be available in container registry at:
 ``my-container-registry.example.com/fedora/rsyslog:32``.
 
 The ``release`` label can also be used to specify the release value use for Koji
-build. When omitted, the release value will be automatically determined by
+build. The value can't be empty, and maybe defined via ENV variable from parent.
```
Comment (athos-ribeiro) [inline docs/users.rst]: s/maybe/may be/

File: docs/users.rst (hunk 1 above)
Comment (rcerven) [reply docs/users.rst]: fixed

File: docs/users.rst (hunk 2 above)
Comment (rcerven) [reply docs/users.rst]: fixed

**Pull request merged**